src/sendRequest/sendRequest.js

```diff
@@ -0,0 +1,8 @@
+const fetch = require('node-fetch')
+
+async function sendRequest(url, reqObj) {
```

Comment:
I prefer to just use promises since async is just a wrapper for a promise no need to further abstract it